app/assets/javascripts/comments.js

```diff
@@ -17,19 +17,31 @@ $(() => {
     if (evt.ctrlKey) { return; }
 
     evt.preventDefault();
-    const $tgt = $(evt.target);
 
-    const resp = await fetch($tgt.attr("href") + '?inline=true', {
+    $tgt = $(evt.target);
```

Comment:
I think this variable can remain as a `const`:

```suggestion
    const $tgt = $(evt.target);
```